app/__init__.py

```diff
@@ -40,4 +40,8 @@ def create_app(db):
 
 db = SQLAlchemy()
 app = create_app(db)
-multiprocessing.set_start_method("fork")
+
+if os.name == 'nt':
```

Comment:
remind me what this is for? what is os.name == 'nt'? can you please add a comment describing this

Reply:
Added comments explaining.